src/application.coffee

```diff
@@ -332,6 +363,122 @@ updateStatus =
 	forceNext: false
 	intervalHandle: null
 
+updateStrategies =
+	'normal-update': (localApp, app, needsDownload, force, timeout) ->
+		Promise.try ->
+			fetch(app) if needsDownload
+		.then ->
+			Promise.using lockUpdates(localApp, force), ->
+				logSystemEvent(logTypes.updateApp, app) if localApp.imageId == app.imageId
```

Comment:
Where this log happens seems to vary between each update strategy, so I'm not sure what stage of an update in general it's meant to signify?